### PR TITLE
Feature/alert message availability #210

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**

### CODE REVIEW

**Review by Erbidium — APPROVED** [review]

**Comment by SlakvichAleksandr** [comment]
https://prnt.sc/GSc5li_fiWY1 - example

File: frontend/src/app/modules/header/header-item/header-item.component.ts [hunk 1]
```diff
@@ -38,32 +38,34 @@ export class HeaderItemComponent extends BaseComponent implements OnInit {
...
         });
 
-        this.teamService.currentTeamEmitted$
-            .pipe(this.untilThis)
-            .subscribe(currentTeamId => {
-                this.teamService.getCurrentUserTeams()
-                    .subscribe(teams => {
-                        this.teams = teams;
+        this.teamService.currentTeamEmitted$.pipe(this.untilThis).subscribe((currentTeamId) => {
+            this.teamService.getCurrentUserTeams().subscribe((teams) => {
```
Comment (tatianahutii) [inline frontend/src/app/modules/header/header-item/header-item.component.ts]: need to unsubscibe

File: frontend/src/app/modules/header/header-item/header-item.component.ts [hunk 2]
```diff
@@ -38,32 +38,34 @@ export class HeaderItemComponent extends BaseComponent implements OnInit {
...
-                            this.teamService.emitCurrentTeamChange(teams[0].id);
-                        } else {
-                            this.currentTeam = this.teams.find(team => team.id === currentTeamId);
-                        }
-                    });
+                if (!currentTeamId && teams.length) {
+                    this.teamService.emitCurrentTeamChange(teams[0].id);
+                } else {
+                    this.currentTeam = this.teams.find((team) => team.id === currentTeamId);
+                }
```
Comment (tatianahutii) [inline frontend/src/app/modules/header/header-item/header-item.component.ts]: move to a separate method

File: frontend/src/app/shared/components/unsaved-changes/unsaved-changes.component.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    title: string;
+
+    message: string;
+
+    constructor(
+        public dialogRef: MatDialogRef<UnsavedChangesComponent>,
+        @Inject(MAT_DIALOG_DATA) public data: IConfirmDialogData,
+    ) {
+        this.title = data.title;
+        this.message = data.message;
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/unsaved-changes/unsaved-changes.component.ts]: why do we need separate properties?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**